app/controllers/api/v1/books_controller.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    class BooksController < ApiController
+      before_action :authenticate_request
+      include Wor::Paginate
+      def index
+        @books = Book.all
+        render_paginated @books, each_serializer: Books::IndexSerializer
+      end
+
+      def show
+        @book = Book.find(params[:id])
```

Comment:
I think you forgot to add `ErrorsHandler` here.
If you look for an book with invalid id, this will throw an exception.